scripts/output/comparison/policyCosts.R

```diff
@@ -120,9 +120,9 @@ write_new_reporting <- function(mif_path,
                                 scen_name,
                                 new_polCost_data) {
 
-  new_mif_path <- paste0(substr(mif_path,1,nchar(mif_path)-4),"_adjustedPolicyCosts.mif")
+  new_mif_path <- mif_path # paste0(substr(mif_path,1,nchar(mif_path)-4),"_adjustedPolicyCosts.mif")
 
-  message("A mif file with the name ", crayon::green(paste0("REMIND_generic_",scen_name,"_adjustedPolicyCosts.mif"))," is being created in the ",scen_name," output folder.")
+  message("A mif file with the name ", crayon::green(new_mif_path), " is written in the ",scen_name," output folder.")
```

Comment:
I guess the message could be changed to something like "The mif file will be overwritten...", but it's just cosmetics :)